src/basic-syntax/static-and-const.md

```diff
@@ -36,4 +38,23 @@ fn main() {
 }
 ```
 
+As noted in the [Rust RFC Book][1], these are not inlined upon use and have an actual associated memory location.  This is useful for unsafe and embedded code, and have a `'static` lifetime[^lifetime].
+
+
 We will look at mutating static data in the [chapter on Unsafe Rust](../unsafe.md).
+
+## Runtime evaluated constants
```

Comment:
I think this will make the page too large. We can move it to a separate page or just let the instructor mention it (`->` speaker notes).

Reply:
Based on your previous note, totally agreed, I'll actually move it to the thread you opened about helpful libraryes

Reply:
Thanks a lot!